panels/source/arrangers/OtherArrangers.js

```diff
@@ -45,9 +45,12 @@ enyo.kind({
 			var f = this.container.toIndex;
 			var c$ = this.getOrderedControls(f);
 			var o = Math.floor(c$.length/2);
+		        var wf = (s === 0 && f == (c$.length-1));
```

Comment:
Does `wf` refer to "forwards" and `wb` refer to "backwards"? If so, should these be reversed? I don't think it makes a difference in the ultimate logic, but more for semantics.